ledger/src/blockstore_processor.rs

```diff
@@ -416,6 +436,7 @@ fn process_entries_with_callback(
     let mut batches = vec![];
     let mut tick_hashes = vec![];
     let mut rng = thread_rng();
+    let cost_model = CostModel::new();
```

Comment:
is there a chance the `cost_model` being shared by threads? Judging `cost_capacity_meter` is protected by RwLock, it might need to do the same for `cost_model`. Perhaps that's why it wasn't shared at this level at first place.

Reply:
It is shared by threads, but it's read-only as far as I could see. Never used mutably.

Reply:
Updated version about to be pushed makes this moot, as the CostModel isn't passed through the threading code any more.